tests/test_arraycreation.py

```diff
@@ -138,9 +138,6 @@ def test_loadtxt(type):
         numpy.testing.assert_array_equal(dpnp_res, np_res)
 
 
-@pytest.mark.parametrize("dtype",
```

Comment:
Please, don't remove your test cases.
Just disable them by adding in skip list.

They may be enabled in the future.